src/js/utils/__tests__/ApplicationUtil-test.js

```diff
@@ -9,44 +9,60 @@ describe('ApplicationUtil', function () {
 
   describe('#invokeAfterPageLoad', function () {
 
+    beforeEach(function () {
+      // Clean up application timers.
+      jasmine.clock().uninstall();
```

Comment:
It some how happens that the after each `uninstall` isn't always working, thus I applied the workaround from  @jfurrow